ansible/roles/eos/tasks/ceos_ensure_reachable.yml

```diff
@@ -0,0 +1,49 @@
+- block:
+  - name: wait for container's mgmt-ip reachable
+    wait_for:
+      port: 22
+      host: "{{ ansible_host }}"
+      search_regex: OpenSSH
```

Comment:
I don't think the `search_regex` argument is important. Recommend to replace it with `state`:

```
      state: started
```

Reply:
done